src/apps/frontend/webpack.dev.js

```diff
@@ -22,7 +22,14 @@ const config = {
     port: devServerPort,
     proxy: {
       secure: false,
-      '/api': `http://localhost:${devServerAPIProxyPort}`,
+      '/api': {
```

Comment:
Can you add please share why this change is needed? My knowledge of webpack is bit dated, rusty at the moment.

Reply:
This `proxy` is being used to forward the requests to the server which contains `/api` in their url. To place the documentation page on this route, we should bypass this specific route. I have added the comments accordingly sir.